requirements/requirements.txt

```diff
@@ -55,7 +55,7 @@ six==1.4.1
 socketpool==0.5.2
 markdown==2.2.1
 django-markup-deprecated
-amqp==1.0.13
+amqp==1.4.6
 amqplib==1.0.2
 sqlagg==0.2.2
 django-redis==3.3
```

Comment:
Just pinning the versions here, all of these came by just doing:
    pip install --upgrade celery
    pip install --upgrade django-celery